### PR TITLE
- Fix commons base flow changes

### PR DESCRIPTION
**Pull request by r0h4n**

### CODE REVIEW

File: tendrl/node_agent/manager/pull_hardware_inventory.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from command import Command
+from tendrl.commons.utils import cmd_utils
```
Comment (nnDarshan) [inline tendrl/node_agent/manager/pull_hardware_inventory.py]: Unless there is a patch to change the current commons. This should be something like:
from tendrl.common.utils import command

File: tendrl/node_agent/manager/pull_hardware_inventory.py [hunk 2]
```diff
@@ -28,7 +28,7 @@ def getNodeCpu():
                    "CoresPerSocket": "corespersocket"}, ...], ...}
 
     '''
-    cmd = Command({"_raw_params": "lscpu"})
+    cmd = cmd_utils.Command({"_raw_params": "lscpu"})
     out, err = cmd.start()
```
Comment (nnDarshan) [inline tendrl/node_agent/manager/pull_hardware_inventory.py]: stdout, stderr, rc = cmd.run() would be the new way to invoke this. Can you please update this

File: tendrl/node_agent/tests/test_ansible_runner.py [hunk 3]
```diff
@@ -6,12 +6,7 @@
 from ansible import modules
```
Comment (nnDarshan) [inline tendrl/node_agent/tests/test_ansible_runner.py]: I think we can delete this file as we no longer have this source in tendrl-node-agent

File: tendrl/node_agent/tests/test_command.py [hunk 4]
```diff
@@ -5,8 +5,8 @@
     import AnsibleExecutableGenerationFailed
```
Comment (nnDarshan) [inline tendrl/node_agent/tests/test_command.py]: this file can also be deleted I feel. As the source for this is deleted

File: tendrl/node_agent/objects/package/atoms/install.py [hunk 5]
```diff
@@ -1,15 +1,12 @@
...
-    import AnsibleRunner
+from tendrl.commons.utils import ansible_module_runner
 
 
 class Install(BaseAtom):
-    def run(self, parameters):
-        name = parameters.get("Package.name")
-        package_type = parameters.get("Package.pkg_type", "pip")
-        version = parameters.get("Package.version", None)
+    def run(self):
```
Comment (nnDarshan) [inline tendrl/node_agent/objects/package/atoms/install.py]: we have instal in the common util as well.
Usage is as follows:
from tendrl.commons.utils import package_installer

in = package_installer.Installer(<package-name>,<package-type(pip/rpm/deb)>,optional<varsion>)
msg, success = in.install(execution_path)

File: tendrl/node_agent/ansible_runner/ansible_module_runner.py [hunk 6]
```diff
@@ -1,81 +0,0 @@
-import ansible.executor.module_common as module_common
```
Comment (nnDarshan) [inline tendrl/node_agent/ansible_runner/ansible_module_runner.py]: There are many other places where Command is being used(like pull_hardware_inventory.py etc..), if you are removing this please change them to use Command from the common utils

File: tendrl/node_agent/manager/pull_hardware_inventory.py (hunk 1 above)
Comment (r0h4n) [inline tendrl/node_agent/manager/pull_hardware_inventory.py]: https://github.com/Tendrl/commons/pull/113

File: tendrl/node_agent/manager/pull_hardware_inventory.py (hunk 2 above)
Comment (r0h4n) [inline tendrl/node_agent/manager/pull_hardware_inventory.py]: Ack

File: tendrl/node_agent/objects/package/atoms/install.py (hunk 5 above)
Comment (r0h4n) [inline tendrl/node_agent/objects/package/atoms/install.py]: ack, will use that

File: tendrl/node_agent/ansible_runner/ansible_module_runner.py (hunk 6 above)
Comment (r0h4n) [inline tendrl/node_agent/ansible_runner/ansible_module_runner.py]: ack

File: tendrl/node_agent/discovery/sds/plugins/discover_ceph_sds_details.py [hunk 7]
```diff
@@ -23,23 +23,28 @@ def discover_storage_system(self):
...
+            # get the cluster_id details
+            cmd = subprocess.Popen(
+                "ceph -s -f json",
+                shell=True,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE
+            )
+            out, err = cmd.communicate()
+            if err:
+                LOG.error("Erro getting cluster details")
```
Comment (shtripat) [inline tendrl/node_agent/discovery/sds/plugins/discover_ceph_sds_details.py]: s/Err/Error/g

**Pull request merged**